README.md

```diff
@@ -29,15 +29,18 @@ class Foo {
   static #p=new Set(), q=Foo.#p;
                 // a private field and a property
                 // of the class constructor                     
-  static get p(){return this.#p} //accessor method     
+  static get p(){return Foo.#p} //accessor method     
   //prototype methods                
   setCallback(f){this.#callback=f}
   constructor(s){
      this.#secret = s;
   }
 }
 ```
-##Details
+
+The most important part of this proposal is the concept of syntactic orthogonality. The key idea is that instead of arbitrarily assigning syntax to various _ClassElement_ constructs, there is a set of simple orthogonal  syntactic units that represent composable semantic concepts. If a JS programmer understand what `static`, #, and the binding list syntactic forms mean then they will understand what `static #foo; ` or `own #bar;` means even if they haven’t specifically been taught about private fields on constructors or own private instance fields.  Orthogonality does introduce some syntactic forms that have limited utility. But that’s ok as the orthogonal consistently makes such rarely used cases understandable. A few special case restrictions is ok, but too many looses the orthogonality and degenerates into a big bag of arbitrary rules.
```

Comment:
On the special case restrictions, it's worth emphasizing, perhaps as early as here, that restrictions -- statically prohibited cases from the matrix -- may surprise authors. But of statically accepted program sources, they cannot confuse readers. The important thing is that all of the matrix that is statically accepted obey non-surprising orthogonality.